paper-slider.html

```diff
@@ -568,6 +572,7 @@
         this._w = this.$.sliderBar.offsetWidth;
         this._x = this.ratio * this._w;
         this._startx = this._x || 0;
+        this._knobstartx = this._startx;
```

Comment:
I believe we can use `_startx` to achieve this behavior. Why introduce a new state?